x-pack/legacy/plugins/monitoring/common/__tests__/format_timestamp_to_duration.js

```diff
@@ -23,22 +23,22 @@ describe('formatTimestampToDuration', () => {
       expect(formatTimestampToDuration(fiftyNineSeconds, CALCULATE_DURATION_SINCE, getTestTime())).to.be('59 seconds');
 
       const fiveMins = getTestTime().subtract(5, 'minutes').subtract(30, 'seconds');
-      expect(formatTimestampToDuration(fiveMins, CALCULATE_DURATION_SINCE, getTestTime())).to.be('5 min');
```

Comment:
The value actually changed?

Reply:
Yeah, 5:30 rounds to 6 minutes now, which seems more accurate to me.